index.js

```diff
@@ -4,18 +4,21 @@ let Promise = require('bluebird')
 let readDir = Promise.promisify(require('recursive-readdir'))
 let fs = Promise.promisifyAll(require('fs'))
 
+let slice = start => thing => thing.slice(start)
+
 // Path Utils
-let filesRelative = (dir, ex) => readDir(dir, ex).map(x => x.slice(dir.length))
+let relativeFilenames = (dir, exclusions) => readDir(dir, exclusions).map(n => slice(dir.length)(n))
```

Comment:
The guide from our onboarding doc is much clearer and JS specific:

https://drboolean.gitbooks.io/mostly-adequate-guide/content/ch5.html#pointfree

You should really read that whole thing, it's pretty quick